app/uk/gov/hmrc/eoricommoncomponent/frontend/views/subscription/sign_in_with_different_details.scala.html

```diff
@@ -44,7 +43,7 @@ <h1 class="heading-large">@messages("cds.subscription.sign-in-with-different-det
 
     <p id="para2">@messages("cds.subscription.sign-in-with-different-details.para2")</p>
 
-    <a href="@logout(service, journey)" class="button" id="sign-out">
+    <a href="@logout(service, journey)" class="button" id="sign-out-button">
```

Comment:
Why change this id?  Does it clash with the one in the header?

Reply:
Yes, accessibility automated test found duplicated id on this page